### DISCUSSION TITLE
G7: yesterday low battery warning, today Sensor Failed 8

### DISCUSSION THREAD

**Discussion by ovinator** [NightscoutFoundation/xDrip#3689] [Bug reports]
Hi, 
anyone encountered this already or has an idea? 
Yesterday the G7 Sensor after 5 days:
"Transmitter battery has dropped to: 276 it may fail soon" failure message in the Android notification section. I ignored it. 
Since this morning voltage is at: A 278, B 213 (no change between 8 am and 5 pm).

Now in the main xdrip window as well as in system status, "Sensor Status: Sensor Failed 8"

No real indication in log.

Just updated to latest version without change. 

Any one am idea what to do? Is it just a bad sensor? 

Samsung A52, recently updated to Android 14 (15 Sep - with the predecessor Sensor that worked well)

**Comment by Navid200** [comment]
It helps if you post a screenshot of the Dex status page.

**Comment by ovinator** [comment]
Yesterday evening:
![threema-20240928-194151035](https://github.com/user-attachments/assets/30aa9d38-5d1a-4b73-9375-e2ad5d5e4849)
This morning:
![threema-20240929-081310291](https://github.com/user-attachments/assets/49ec0eef-541e-4b80-9ce1-66f9bac9dcac)
This afternoon:
![threema-20240929-173501402](https://github.com/user-attachments/assets/46a5e08a-abcb-4222-8c29-f42a4f897125)
![threema-20240929-173725084](https://github.com/user-attachments/assets/16141919-8b98-48c6-8473-7f3861f67ad2)
![threema-20240929-173910598](https://github.com/user-attachments/assets/05ccf985-ff41-4506-9f7b-b849e718e580)

Here you are.

**Reply by Navid200** [reply]
Can you provide a screenshot of the battery voltage logs?
https://navid200.github.io/xDrip/docs/Dexcom/BatteryVoltageLogs.html

**Comment by ovinator** [comment]
![threema-20240929-193821721](https://github.com/user-attachments/assets/ce894c92-d39a-4a04-84ac-3018764c0df1)

**Reply by Navid200** [reply]
That only covers 4 days.  Is there a reason you are not showing all the 8 days?

**Reply by ovinator** [reply]
Keyboard was hiding the rest, updated

**Reply by ovinator** [reply]
Ok, a brand new one 
![threema-20240929-194708986](https://github.com/user-attachments/assets/d75682d8-5525-420d-8d46-34f2b355e7ef)

**Reply by Navid200** [reply]
This is what it looks like:
![Screenshot 2024-09-29 140555](https://github.com/user-attachments/assets/da5b0ab2-0be4-4b8a-a95e-253f58db2dfc)
<br/>    

What are the two dates on the sensor box?

**Reply by ovinator** [reply]
1.4.2024 / 30.9.2025

**Reply by Navid200** [reply]
Then, you can try Dexcom app after disabling collection in xDrip: https://navid200.github.io/xDrip/docs/Stop-xDrip.html 
If it doesn't work, you can contact Dexcom and ask for advice.

**Reply by ovinator** [reply]
We started another Sensor in parallel. Who is short before end of warning up. When I disable collection - it might break the new one. 
Can't I install Dexcom App on another phone without stopping collection

**Reply by Navid200** [reply]
You can install Dexcom app on another phone regardless of what you are doing on your main phone.  You don't have to disable collection in xDrip if you are using Dexcom app for another sensor on another phone.

**Reply by ovinator** [reply]
Thanks a lot for your support.

**Reply by ovinator** [reply]
Can I donate to a thank you box in paypal or so?

**Reply by Navid200** [reply]
Thanks
It's great to know that you are happy.  That is rewarding enough.

**Reply by WesSec** [reply]
> 1.4.2024 / 30.9.2025

Damn, was just researching why i'm getting a sensor failed 8 error and my dates match exactly yours, guess it was a bad batch

**Reply by Navid200** [reply]
The only way to know if something is wrong with G7 is to use Dexcom app for verification.
There is no way to be sure just because something does not work with xDrip.

**Reply by WesSec** [reply]
I tried connecting it to the G7 app on another phone, but it wouldn't even connect anymore, probably the battery is depleted

**Reply by Navid200** [reply]
That is a possibility as long as you have checked with a scanner to make sure it is not transmitting: https://navid200.github.io/xDrip/docs/Bluetooth-Scanner.html
If you see that it is not transmitting after having been inserted, you should use the magnet trick:
https://navid200.github.io/xDrip/docs/Dexcom/G7_Troubleshooting.html 
If that doesn't fix it, I guess it has to be damaged.

**Reply by WesSec** [reply]
just to clarify and make sure we're on the same page, the sensor was working fine for 5 days, the battery level was reported low, but did not have a significant decrease the last logs, i called dexcom and they replaced it without issue, i just found it interesting that this sensor is probably from the same batch as reported by OP :)
